### DISCUSSION TITLE
ICRS grid in Stellarium 0.21.0 and 0.21.1

### DISCUSSION THREAD

**Discussion by ruswafalak** [Stellarium/stellarium#1767] [Q&A] [answered]
ICRS grid in Stellarium 0.21.0 and 0.21.1 is missing. Is it not implementation anymore? Why?

**Comment by gzotti** [comment, marked as answer]
You may have to enable the button. F2/Extras

**Reply by ruswafalak** [reply]
Ok Thank you
